src/plugins/workspace/server/plugin.ts

```diff
@@ -92,7 +93,11 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
 
     core.savedObjects.setClientFactoryProvider(
       (repositoryFactory) => ({ includedHiddenTypes }: { includedHiddenTypes?: string[] }) =>
-        new SavedObjectsClient(repositoryFactory.createInternalRepository(includedHiddenTypes))
+        new SavedObjectsClient(
+          repositoryFactory.createInternalRepository([
+            ...new Set([WORKSPACE_TYPE, ...(includedHiddenTypes || [])]),
```

Comment:
Will this a little bit strange? We can't exclude workspace type anymore, if we don't want to include this type.

Reply:
Yeah, let me refactor this part, create new client inside the wrapper.